src/components/InstancesList/InstancesList.js

```diff
@@ -847,7 +847,7 @@ class InstancesList extends React.Component {
           anchor = records[0].name;
         }
 
-        source.fetchByQuery(`${param} < "${anchor}"`);
+        source.fetchByQuery(`${param} < "${anchor.replace(/"/g, '')}"`);
```

Comment:
This will just remove quotes, and ticket description mentions escaping
Should it be `anchor.replace(/"/g, '\\"')` to replace `"` with `\"` instead?

Reply:
@BogdanDenis with this replace it's work not how we expect. So I don't think it will be better.